dts/bindings/can/st,stm32-can.yaml

```diff
@@ -20,6 +20,14 @@ properties:
     pinctrl-names:
       required: true
 
+    one-shot:
+      type: boolean
+      required: false
```

Comment:
```suggestion
      required: false
      default: false
```

Reply:
Leaving out a boolean dts property is the same as setting it to false. No need for `default`.

Reply:
Good to know! Thanks!